scripts/nodejs/server.js

```diff
@@ -54,6 +54,26 @@ function genTemplate(filePath, packageName, callback) {
     });
 }
 
+function listTupaiClass(callback) {
+    var packagejsHtml = '<script src="__tupairoot/libs/package.js"></script>';
+    var tupaiSrcDir = path.join(__dirname, '..', '..', 'src', 'tupai');
+    execute(['list', '--classPath', tupaiSrcDir, '--ignoreNotFound'], function(output) {
+        //if (!output) {return;}
+        var classes = JSON.parse(output);
+        //console.log(classes);
+        var tupaijsfiles = classes.map(function(cls){
+            var path = cls.path;
+            return path.replace(tupaiSrcDir + "/", "");
+        });
+
+        var scripts = tupaijsfiles.map(function(filename){ return '<script src="__tupairoot/src/tupai/' + filename + '"></script>';});
```

Comment:
It's kind of Unix Philosophy. "Do one thing".
the purposes of 2 maps are different.
First map is to **remove** extra information from the output of list command.
Second map is to **add** required stirngs to compose HTML `script` element.